_glaredb/sql-commands/copy-to.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: COPY TO
+parent: SQL commands
+---
+
+# COPY TO
+
+Copy a query result to a file.
+
+GlareDB currently only supports creating local copies using this command.
+
+## Syntax
+
+```sql
+COPY (<query>) TO LOCAL [FORMAT <format>] (location <path>);
```

Comment:
Apart from `local`, we do support `gcs` and `s3`.

GCS requires additional `bucket` and `service_account_key` options.

S3 requires additional `bucket`, `region`, `access_key_id` and `secret_access_key` options.

You can also pass in credentials for both of above. GCP credentials for GCS and AWS for S3.

Reply:
After QAing it, there are some bugs still, so we'll document it next release.